lib/reducers/create-otp-reducer.js

```diff
@@ -222,7 +228,13 @@ function createOtpReducer(config) {
   validateInitialState(initialState)
 
   /* eslint-disable-next-line complexity */
-  return (state = initialState, action) => {
+  return (prevState = initialState, action) => {
+    // Update last action timestamp for eligible actions.
+    const state = TIMEOUT_IGNORED_ACTIONS.includes(action.type)
```

Comment:
Can eligible actions be defined in a config, or are they the same for every deployment? Not sure if that would be important, but maybe different agencies would want to allow different actions to reset the timer

Reply:
They will be the same for all deployments that use the optional timeout feature as this is trying to catch actions initiated by the user.